src/style.toit

```diff
@@ -423,7 +423,7 @@ class RoundedCornerOpacity_:
   radius/int
   static cache_ := Map.weak
 
-  static OPAQUE-BYTES-8x8_/ByteArray ::= ByteArray 64: 0xff
+  static OPAQUE-BYTES-16x16_/ByteArray ::= ByteArray 256: 0xff
```

Comment:
I'm pretty sure I asked this before, but just didn't see the response: why is this not a literal?

Reply:
I can't find that.  I think may be thinking of.
https://github.com/toitware/toit-pixel-display/pull/55/files/0d382646d2b146089bd6405b64958666b2af96db#r1417165258

Reply:
In the end I decided not to make this change, because it harmonizes badly with the request to use a constant for the patch size.  There's no way to make a literal that is sized based on a different constant.